docs/code/reference.md

```diff
@@ -117,6 +117,13 @@ than good.
         l = n * 2 + 4;   // Clear l variable
     }
 
+For translators' comments, use triple forward slash `///`.
+```
+{
+    "elementary OS": false /// Can't be translated
```

Comment:
This should probably be `/// Shouldn't be translated because it is a brand name.`